### PR TITLE
Feat: Added Blog for the project

### PR DESCRIPTION
**Pull request by Jaiharishan**
Worked on creating the blog page. (ISSUE #11 )

### CODE REVIEW

**Comment by Hiccup1234** [comment]
looking at it isn't contribution guidlines and setup markdown files overlapping a bit?. We can make it one blog in my opinion

File: docusaurus.config.js [hunk 1]
```diff
@@ -1,53 +1,55 @@
...
-  url: 'https://painite.nitt.edu',
-  baseUrl: '/',
-  onBrokenLinks: 'throw',
-  onBrokenMarkdownLinks: 'warn',
-  favicon: 'img/favicon.ico',
+  title: "Painite",
+  tagline:
+    "Painite project - to-go website for nitt students for all academic infomations.",
+  // url: "https://painite.nitt.edu",
+  url: "http://192.168.1.3:3000/",
```
Comment (jaiakash) [inline docusaurus.config.js:13]: This url is kept when we deploy the website. Are you sure we should keep the local URL as website. Let me know what you think.

File: docusaurus.config.js [hunk 2]
```diff
@@ -130,6 +132,20 @@ const config = {
         theme: lightCodeTheme,
         darkTheme: darkCodeTheme,
       },
+      algolia: {
```
Comment (jaiakash) [inline docusaurus.config.js:135]: Nice, :1st_place_medal: Is Algolia for search?

**Review by jaiakash — APPROVED** [review]

**Comment by jaiakash** [comment]
Closes #18

**Comment by jaiakash** [comment]
@Jaiharishan Can you please fix the merge conflicts.

File: docusaurus.config.js (hunk 2 above)
Comment (Jaiharishan) [reply docusaurus.config.js:135]: Yes, but currently, it is not functional. It will be available after hosting.

**Comment by Jaiharishan** [comment]
> @Jaiharishan Can you please fix the merge conflicts.

Yeah ok.

File: docusaurus.config.js (hunk 1 above)
Comment (Jaiharishan) [reply docusaurus.config.js:13]: I kept the local URL for the time being. We need to change it once we deploy.

**Pull request merged**